server/src/main/java/io/druid/segment/realtime/RealtimeMetricsMonitor.java

```diff
@@ -80,7 +80,9 @@ public boolean doMonitor(ServiceEmitter emitter)
       );
       emitter.emit(builder.build("ingest/persists/failed", metrics.failedPersists() - previous.failedPersists()));
       emitter.emit(builder.build("ingest/handoff/failed", metrics.failedHandoffs() - previous.failedHandoffs()));
-
+      emitter.emit(builder.build("ingest/merge/time", metrics.mergeTimeMillis() - previous.mergeTimeMillis()));
+      emitter.emit(builder.build("ingest/merge/cpu", metrics.mergeCpuTime() - previous.mergeCpuTime()));
+      emitter.emit(builder.build("ingest/persist/cpu", metrics.persistCpuTime() - previous.persistCpuTime()));
```

Comment:
also, it appears we will be publishing these metrics whether or not cpu time is supported , so should document that if user sees a value of 0 then that means its just not supported on the jvm